config/prow/cluster/BUILD.bazel

```diff
@@ -73,7 +73,11 @@ release(
     component("prow-canary-k8s-io", "managedcertificate"),
     component("prow-k8s-io", "managedcertificate"),
     component("prow-kubernetes-io", "managedcertificate"),
-    component("prowjob", "customresourcedefinition"),
+    component(
+        "prowjob",
+        "customresourcedefinition",
+        args = ["--server-side=true"],
```

Comment:
nit: we're just speculating that this will work, but not sure. might suggest move this change to a separate PR just in case revert is needed we don't need to revert this large PR

Reply:
Well, not doing this will definitely not work as the PR changes the CRD file :grimacing:

Reply:
well, finger crossed then